src/op_mode/generate_ssh_server_key.py

```diff
@@ -17,10 +17,15 @@
 from sys import exit
 from vyos.util import ask_yes_no
 from vyos.util import cmd
+from vyos.util import commit_in_progress
 
 if not ask_yes_no('Do you really want to remove the existing SSH host keys?'):
     exit(0)
 
+if commit_in_progress():
+    print('Can not restart ssh.service until commit in progress')
```

Comment:
Just use SSH - no more, no less.

Reply:
Also, s/can not/cannot/

Reply:
...and s/until commit in progress/while a commit is in progress/

Reply:
@c-po @dmbaturin Which one is better to use?

```
Cannot clear DHCP lease as a commit is still in progress
```
```
Cannot clear DHCP lease while a commit is in progress
```